primo/data_parser/input_options_well_data.py

```diff
@@ -0,0 +1,214 @@
+#################################################################################
+# PRIMO - The P&A Project Optimizer was produced under the Methane Emissions
+# Reduction Program (MERP) and National Energy Technology Laboratory's (NETL)
+# National Emissions Reduction Initiative (NEMRI).
+#
+# NOTICE. This Software was developed under funding from the U.S. Government
+# and the U.S. Government consequently retains certain rights. As such, the
+# U.S. Government has been granted for itself and others acting on its behalf
+# a paid-up, nonexclusive, irrevocable, worldwide license in the Software to
+# reproduce, distribute copies to the public, prepare derivative works, and
+# perform publicly and display publicly, and to permit others to do so.
+#################################################################################
+
+"""
+This module defines all the input options for objects of WellData class
+"""
+
+
+# Installed libs
+from pyomo.common.config import (
+    Bool,
+    ConfigDict,
+    ConfigValue,
+    In,
+    NonNegativeFloat,
+    NonNegativeInt,
+)
+
+# User libs
+from primo.utils.domain_validators import InRange
+from primo.data_parser.default_data import SUPP_IMPACT_METRICS
+
+
+# This dictionary contains default values to fill with,
+# if the data is not provided.
+binary_type_metrics = {
+    "leak": True,  # Well is leaking
```

Comment:
If the data is not provided, we consider the well to not have a leak. In that case, this should be False right?

Reply:
Yeah. I thought we're assuming that the well is leaking if it is not provided. I'll change it to False.